UnityAds/Video/UADSAVPlayer.m

```diff
@@ -123,6 +123,7 @@ - (void)stopPrepareTimeoutTimer {
 
 - (void)onPrepareTimeoutListener:(NSNotification *)notification {
     UADSLogError(@"Video prepare timeout");
+    self.prepareTimeoutTimer = NULL;
```

Comment:
Crash will be, when will be called `- (void)onPrepareTimeoutListener:(NSNotification *)notification` by timer. And later will be called `stopPrepareTimeoutTimer()` function.